rotkehlchen/db/snapshots.py

```diff
@@ -109,26 +116,53 @@ def export(self, timestamp: Timestamp, directory_path: Optional[Path]) -> Tuple[
             return self.create_zip(
                 timed_balances=timed_balances,
                 timed_location_data=timed_location_data,
+                main_currency=main_currency,
+                timestamp=timestamp,
             )
 
         return self._export(
             timed_balances=timed_balances,
             timed_location_data=timed_location_data,
             directory=directory_path,
+            main_currency=main_currency,
+            timestamp=timestamp,
         )
 
+    @staticmethod
+    def _convert_currency(serialized_data: List[Dict[str, str]], main_currency: Asset, timestamp: Timestamp) -> None:  # noqa: E501
```

Comment:
It's not wrong what you did here but you could move this to being a module level helper method, since it's not used anywhere else and you can't call it with `self` like you can do with `_export` since it has no `self`. 

Kind of a nitpick but would fit the way we do things more.